### DISCUSSION TITLE
Creating a password cracking dictionary

### DISCUSSION THREAD

**Discussion by danielhw2** [sepinf-inc/IPED#866] [Q&A] [answered]
Olá.

Qual a opção preciso utilizar para que o IPED crie um dicionário para ser utilizado na quebra de senhas, a partir de uma imagem Encase? 
Qual seria a linha de comando?

**Comment by wladimirleite** [comment, marked as answer]
Not sure if that is what you are looking for, but there is an option to export all indexed terms ("Exportar lista de termos indexados" in Portuguese, located in the main options  menu).
You have to process one or more evidences in the "usual way", then open the processed case and finally export the list of words.

Two important details:
- **Case distinction**: usually indexed terms are converted to lowercase before indexing, so case information will be lost when exporting the word list. If you would like to keep uppercase/lowercase letters exactly as they were found, there is a parameter in the **AdvancedConfig.txt** file named **convertCharsToLowerCase**. Default is true, and it should be changed to false.
- **Special characters**: only letters and numbers are indexed by default. Other characters are considered delimiters. This behavior can also be changed in the same configuration file, using **extraCharsToIndex** entry. For example, asterisk and underscore could be added.

A last tip, before processing the actual evidence file, after changing the configuration file, make a quick test processing a single text file and then exporting the word list, so you can check if the result is the expected.

**Reply by danielhw2** [reply]
Thank you very much!
